atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -219,7 +222,7 @@ def run(self):
             top_layer, _ = pulp_push_results
             annotations['v1-image-id'] = top_layer
 
-        pulp_pull_results = self.workflow.postbuild_results.get(PulpPullPlugin.key)
+        pulp_pull_results = self.workflow.exit_results.get(PulpPullPlugin.key)
```

Comment:
No, this is specifically for arrangement versions < 4 which ran pulp_pull as a post-build plugin in the worker build. It could definitely use a comment though.

Reply:
+1 for adding a comment

Reply:
`pulp_pull_results` would now check both postbuild and exit plugins lists. Added a comment